### PR TITLE
Simple get single user unauthorized implementation

### PR DESCRIPTION
**Pull request by sergiitk**
Simple implementation of an unauthorized call to Northstar to get user by id.

Tests:

```
  ․․․

  3 passing (613ms)
```

### CODE REVIEW

**Comment by sergiitk** [comment]
CC @DFurnes as usual, please take a look at it if you got a minute.

File: lib/northstar-client.js [hunk 1]
```diff
@@ -10,12 +12,55 @@ const VERSION = require('../package.json').version;
 class NorthstarClient {
 
   constructor(options) {
-    this.options = options;
+    this.options = options || {};
     this.VERSION = VERSION;
+
+    // TODO: actually parse options.
+    this.options.baseURI = 'https://northstar-qa.dosomething.org/v1/users';
```
Comment (deezone) [inline lib/northstar-client.js:19]: Should this be in a config file, out of the repo? Perhaps support for ENV flags for test vs production as I suspect production will not want to point at `northstar-qa`?

Comment (sergiitk) [inline lib/northstar-client.js:19]: Yep, that's why there's a TODO comment above 😄

**Comment by deezone** [comment]
👍

**Pull request merged**